src/config.rs

```diff
@@ -2496,9 +2496,6 @@ pub struct BackupStreamConfig {
     pub initial_scan_pending_memory_quota: ReadableSize,
     #[online_config(skip)]
     pub initial_scan_rate_limit: ReadableSize,
-    #[serde(skip)]
-    #[online_config(skip)]
-    pub use_checkpoint_v3: bool,
```

Comment:
Will it cause compatibility issue?

Reply:
In all release version this field should be `#[serde(skip)]`, I guess this would be fine.🤔